Formula/mariadb@10.1.rb

```diff
@@ -24,7 +24,6 @@ class MariadbAT101 < Formula
   deprecated_option "with-tests" => "with-test"
 
   depends_on "cmake" => :build
-  depends_on "pidof" unless MacOS.version >= :mountain_lion
```

Comment:
That change is `if MacOS.version >= :mountain_lion` which means pidof would still be need for < :mountain_lion unless I misunderstood something.